pinc/smoothread.inc

```diff
@@ -283,7 +283,7 @@ function handle_smooth_reading_change($project, $postcomments, $days, $extend)
         SET $smoothread_deadline_sql
             postcomments = CONCAT(postcomments, '%s')
         WHERE projectid = '$projectid'
-    ", mysqli_real_escape_string(DPDatabase::get_connection(), $postcomments));
+    ", mysqli_real_escape_string(DPDatabase::get_connection(), html_entity_decode($postcomments)));
```

Comment:
It sounds like this was hashed out in the Slack discussion, but we only need `html_entity_decode()` in the update script to fix problematic database entries. [This forum post](https://www.pgdp.net/phpBB3/viewtopic.php?f=16&t=16898&p=1197231#p1197231) discusses how they got in there, why we need to convert them, and why we don;t have to worry about this after we switch to Unicode.